ocaml/idl/datamodel_cluster_host.ml

```diff
@@ -91,6 +91,17 @@ let forget = call
   ~hide_from_docs:true
   ()
 
+let resync_host = call
```

Comment:
I don't think we need to expose this API, we got pool_resync already, see below.

Reply:
we do because `pool_resync` needs to be able to call `resync_host` on the various hosts. it does not need a host parameter though, which is good

Reply:
Interesting, how did this ever work before then? Was it just silently resyncing the wrong host?

Reply:
I think we just never tested `xe cluster-pool-resync` thoroughly enough before, lets add the new API call then.

Reply:
i think it must have been, but we probably didn't notice because it was still making the enable rpc call